content/en/docs/application-store/app-lifecycle-management.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "App Lifecycle Management"
+keywords: 'kubernetes, kubesphere, app-store'
+description: 'App Lifecycle Management'
+
+
+weight: 2240
+---
+
+KubeSphere integrates open source [OpenPitrix](https://github.com/openpitrix/openpitrix) to set up app store and app repository services which provide full lifecycle of application management. Application Store supports three kinds of application deployment as follows:
+
+> - **Application template** provides a way for developers and ISVs to share applications with users in a workspace. It also supports importing third-party application repositories within workspace.
+> - **Composing application** means users can quickly compose multiple microservices into a complete application through the one-stop console.
+
+![App Store](/images/application-templates/app-store.png)
+
+## Objective
+
+In this tutorial, we will walk you through how to use [EMQ X](https://www.emqx.io/) as a demo application to demonstrate the **global application store** and **application lifecycle management** including upload / submit / review / test / release / upgrade / delete application templates.
+
+## Prerequisites
+
+- You need to install [Application Store (OpenPitrix)](../../installation/install-openpitrix).
+- You need to create a workspace and a project, see [Get Started with Multi-tenant Management](../admin-quick-start).
```

Comment:
Both links do not open properly and need to be modified or replaced with a different text description

Reply:
fixed